generated/nidaqmx/constants.py

```diff
@@ -57,8 +57,8 @@ class AccelUnits(Enum):
 
```

Comment:
I think the inconsistencies in this file highlights why we want to generate nidaqmx-python and nidaqmx.proto from the same metadata in the first place: otherwise, it will be difficult to match up things that have different names in Python vs. gRPC.

I also think that a lot of these differences are caused by using different parts of the metadata: enum tFoo vs. lvEnum Foo1, etc. Some are also caused by the Python-specific fixups and substitutions (or lack thereof).

Reply:
Sure Brad. Most of inconsistencies are resolved and with the updated enums metadata, we have the changes in `constants.py` categorized as mentioned in this https://github.com/ni/nidaqmx-python/pull/215#issuecomment-1468081159